appium/api-mocks/apis/google/exported-messages/message-export-18481ad047f62d2a.json

```diff
@@ -1,18 +1,21 @@
 {
   "acctEmail": "e2e.enterprise.test@flowcrypt.com",
   "full": {
-    "id": "18481ad047f62d2a",
-    "threadId": "18481ad047f62d2a",
-    "labelIds": ["IMPORTANT", "CATEGORY_PERSONAL", "INBOX"],
-    "snippet": "-----BEGIN PGP MESSAGE----- Version: FlowCrypt Email Encryption 8.3.8 Comment: Seamlessly send and receive encrypted email wV4DT2ZlSmhZ1GoSAQdA22tinAfivRH6BmMpKO/LI923FDu0+02DQ3fmK0PX XBYwMIOo+",
+    "id": "18481acfce07949c",
+    "threadId": "18481acfce07949c",
+    "labelIds": [
+      "SENT",
+      "INBOX"
+    ],
+    "snippet": "",
     "payload": {
       "partId": "",
-      "mimeType": "multipart/mixed",
+      "mimeType": "multipart/encrypted",
```

Comment:
as I remember, `multipart/mixed` mime type was set by gmail api (https://github.com/FlowCrypt/flowcrypt-browser/issues/3157#issuecomment-731027045) for pgp/mime messages.
is this new message json from real rich text message?

I wanted to re-test it, but rich text message sending fails for me in firefox - https://github.com/FlowCrypt/flowcrypt-browser/issues/5903

Reply:
I downloaded it (new message json) from compatibility account with our flowcrypt `download api export` feature.